src/FixerFactory.php

```diff
@@ -192,4 +208,51 @@ private function sortFixers()
 
         return $this;
     }
+
+    /**
+     * @param FixerInterface $fixer               fixer to test
+     * @param string[]       $candidateFixerNames fixer names to test against
+     *
+     * @return string[]|null
+     */
+    private function getConflictingFixers(FixerInterface $fixer, array $candidateFixerNames)
+    {
+        $conflictMap = $this->getFixersConflicts($fixer); // to be replaced in the future by calling the Fixer itself for its conflicts
+        if (null === $conflictMap) {
+            return;
+        }
+
+        $conflictingFixers = array();
+        foreach ($candidateFixerNames as $candidate) {
```

Comment:
I would iterate over `conflcitMap` instead of `candidateFixerNames`, first one will almost always be smaller